### PR TITLE
doc: add reporting bugs doc and small nit improvements in the README

### PR DESCRIPTION
**Pull request by camilamacedo86**
doc: add reporting bugs doc and small nit improvements in the README

### CODE REVIEW

**Comment by camilamacedo86** [comment]
c/c @njhale

**Comment by camilamacedo86** [comment]
@njhale wdyt about this one? is it ok to be merged? Could you please give a hand with?

**Comment by Bowenislandsong** [comment]
Thanks for doing this.
/lgtm

**Comment by camilamacedo86** [comment]
Hi @Bowenislandsong, 

I rebased it with the master again to see if the CI issue is fixed.

File: README.md [hunk 1]
```diff
@@ -56,6 +58,18 @@ $ make run-console-local
 
 Then visit `http://localhost:9000` to view the console.
 
+## End-to-end example
+
+For a complete end-to-end example of how OLM fits into the Operator Framework, see the [Operator Framework Getting Started Guide](https://github.com/operator-framework/getting-started). Also, see [Getting Started on OperatorHub.io](https://operatorhub.io/getting-started).
```
Comment (njhale) [inline README.md:63]: The sentence here is duplicated above.

**Comment by camilamacedo86** [comment]
Hi @njhale,  

Could we move with this one now?

File: README.md (hunk 1 above)
Comment (camilamacedo86) [inline README.md:63]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ Install OLM on a Kubernetes or OpenShift cluster by following the [installation
 
 For a complete end-to-end example of how OLM fits into the Operator Framework, see the [Operator Framework Getting Started Guide](https://github.com/operator-framework/getting-started).
```
Comment (camilamacedo86) [inline README.md:45]:
```suggestion
For a complete end-to-end example of how OLM fits into the Operator Framework, see the [Operator Framework Getting Started Guide](https://github.com/operator-framework/getting-started). Also, see [Getting Started on OperatorHub.io](https://operatorhub.io/getting-started).
```

File: README.md (hunk 1 above)
Comment (camilamacedo86) [reply README.md]: great catcher.

**Comment by camilamacedo86** [comment]
/test e2e-gcp-upgrade
/test e2e-aws-olm
/test e2e-gcp

@ecordell @njhale @Bowenislandsong I already rebased with the master and re-started this test a lot of times. It is just a change in the README. In POV should not call the jobs indeed since is a just a doc. however, I do not know why it does not pass in the CI.

**Pull request merged**